templates/registry/registry-dpl.yaml

```diff
@@ -44,6 +44,16 @@ spec:
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      initContainers:                                                           
+      - name: "change-permission-of-directory"                                  
+        image: {{ .Values.registry.registry.initContainerImage.repository }}:{{ .Values.registry.registry.initContainerImage.tag }}
+        imagePullPolicy: {{ .Values.imagePullPolicy }}                          
+        command: ["/bin/sh"]                                                    
+        args: ["-c", "chown -R 10000:10000 /storage"]                           
```

Comment:
Please don't hardcode the `/storage` location here. You should use the values.yaml parameter value `{{ .Values.persistence.imageChartStorage.filesystem.rootdirectory }}` instead.
You can also use `harbor:harbor` instead of `10000:10000` if you reuse the `harbor-registry` or `harbor-registryctl` image.